src/components/validators/ValidatorDataTable.vue

```diff
@@ -0,0 +1,159 @@
+      if (alpha2) {
+        return `flag-icon-${alpha2}`;
+      }
+      return '';
+    }
+
+    function updateVote(val: string[]) {
+      store.commit('account/setVote', val);
+    }
+
+    function isTop21(val: string): boolean {
+      return bpTop21.value.includes(val);
+    }
+
+    onMounted(() => {
+      console.log(producerRows.value);
+      console.log(account.value);
+    });
+    return {
+      producerRows,
+      account,
+      HeadProducer,
+      removeVote,
+      selection,
+      maxSelected,
+      projectedVoteWeight,
+      weightChange,
+      tableHeader,
+      getLink,
+      getFlag,
+      currentVote,
+      pagination,
+      updateVote,
+      producerPay,
+      isTop21
+    };
+  }
+});
+</script>
+
+<template lang="pug">
+.q-pa-md
+  div.bp-list(style="overflow-x: scroll; width: 100%;")
+    .row.q-col-gutter-sm
+      .col-12(style="min-width: 1000px;")
+        q-card(flat)
+          .q-card-section.q-pa-md.text-subtitle1.text-weight-light
+            .row
+              .col-1 Rank
+              .col-3  BP
+              .col-2.offset-1  Rank
+              .col-2  Votes
+              .col-2  Reward 24h
+      .col-12( v-for='(bp,i) in producerRows' style="min-width: 1000px;")
+        q-card.producer-card(flat)
+          .q-card-section
+            .row
+              .col-1.q-py-md
+                .row.items-center.full-height.text-h6.q-px-md {{producerRows.indexOf(bp) + 1}}
+              .col-3.q-py-md
+                .text-uppercase.text-h6 {{ bp.org?.candidate_name || bp.owner }}
+                .text-body2 {{ bp.org?.location?.name ? bp.org.location.name : 'unknown' }}
+              .col-2.q-py-md.offset-1
+                .row.items-center.full-height
+                  q-chip(v-if="HeadProducer === bp.owner " square color="primary" text-color="white" label="Producing")
+                  q-chip(v-else-if="(producerRows.indexOf(bp) + 1) < 22" outline square color="primary" text-color="white" label="Top 21")
+                  q-chip(v-else-if="(producerRows.indexOf(bp) + 1) < 43" outline square color="primary" text-color="white" label="Standby")
+                  q-chip(v-else outline square color="primary" text-color="white" label="Unpaid Standby")
+              .col-2.q-py-md
+                .row.items-center.full-height {{ (bp.total_votes / 10000).toLocaleString(undefined, {minimumFractionDigits: 4,maximumFractionDigits: 4,}) }}
+              .col-2.q-py-md
+                .row.items-center.full-height {{ ((producerRows.indexOf(bp) + 1) < 22 ? producerPay : (producerRows.indexOf(bp) + 1) < 43 ? producerPay / 2 : 0 ).toFixed(0)  + ' TLOS' }}
+              .col-1.select-box.q-py-md
+                .row.full-selection.justify-center
+                  q-checkbox(v-model="currentVote" :val="bp.owner" @update:model-value="(val)=> updateVote(val)")
+
+</template>
+
+<style lang="sass" scoped>
+.producer-card
+  background: #f5f4fe
+.select-box
+  background: #e0dffb
```

Comment:
move hex into the css variables file and reference as a variable here so we can keep track of the colors we are using throughout the app